dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -752,9 +752,9 @@ def generate_provider_dependencies_if_needed():
 ]
 
 ALL_PYTHON_VERSION_TO_PATCH_VERSION: dict[str, str] = {
-    "3.6": "v3.6.1",
-    "3.7": "v3.7.1",
-    "3.8": "v3.8.1",
+    "3.6": "v3.6.15",
+    "3.7": "v3.7.17",
+    "3.8": "v3.8.20",
```

Comment:
What are we testing in CI against 3.6-3.8 that this matters?

Reply:
I don't think these are used in CI either, as even 3.9 is deprecated in the repository for Airflow. However, the CI failure is why I updated them. Perhaps more cleanup is required if we don't check these ones?

Reply:
It is seen in pre commit as a change, but let's merge UV update and see it again

Reply:
It was because I fixed GITHUB_TOKEN in upgrade_check. Previously it failed because of rate limits and it did not upgrade the python version.

Reply:
And yes - we could likely remove them altogether - we do not need 3.6/3.7/3.8 - we still need 3.9 because when this script is cherry-picked to`v3-0-test` it should also check Python 3.9 (though technically there should be no more releases of 3.9).